### PR TITLE
Remove fluff from TOC

### PR DESCRIPTION
**Pull request by sosiouxme**
I would assert that the purpose of a TOC is not to pedantically list out every heading. It's to help readers notice or find stuff. Anything that doesn't serve that purpose is working against it as it clutters the TOC. So while it's great to auto-generate the headings for accuracy, I think we should then prune any TOC entry that doesn't do one of:
1. Help the first-time reader get an overview of what the training involves
2. Help a reader with a single interest (e.g. "does it do quotas? how?") find it in the document
3. Help a user find their place again when returning to it
4. Direct to specific tasks after training completion -- act as reference

Basically, something people will actually look for. So for instance, we don't need a TOC entry for every "create a project" and "look at your app in the web console".

I'd assert the pruning is worthwhile... but leaving this as a PR for discussion.

### CODE REVIEW

**Comment by detiber** [comment]
@sosiouxme perhaps the correct approach for this is to not use markdown headings for the sections we want to prune and to instead just bold/underline them or distinguish them in some other way than a heading?  That would remove the guess work required from having to prune the TOC after generation.

**Comment by sdodson** [comment]
Could we not just demote those items to #### headers so they don't show up?

**Comment by sosiouxme** [comment]
I'd prefer Jason's suggestion of not using headings for sections that don't belong in TOC. They aren't necessarily sub-headings structurally, they are just sort of breaks and don't contribute to the TOC.

**Comment by sdodson** [comment]
:+1:

**Comment by thoraxe** [comment]
I think we should either turn down the doctoc to level 2 or move
non-essential pauses to level 4.

**Comment by thoraxe** [comment]
Soooo... any objections to the above?

I am going to close this and convert it to an issue and ask people to identify non-essential headings.
